views/pages/camps/index_admin.jade

```diff
@@ -78,11 +82,12 @@ block content
                                 a(ng-href='{{camp.facebook_page_url}}', target='_blank')
                                     button.btn.btn-xs
                                         span(class='fa fa-facebook-official')
-                                a.cursor-p(ng-href=`/${language}/camps/{{camp.id}}` + '/edit')&nbsp;
-                                    button.btn.btn-xs 
-                                        span(class='glyphicon glyphicon-pencil')
-                                        =t('camps:stats.edit')
-                                a.cursor-p(ng-click='removeCamp(camp.id)')&nbsp;
+                                if(user.isAdmin || (user.isCampsAdmin && (new Date() < controllDates.edit_camps_lastDate)))
+                                    a.cursor-p(ng-href=`/${language}/camps/{{camp.id}}` + '/edit') &nbsp;
+                                        button.btn.btn-xs 
+                                            span(class='glyphicon glyphicon-pencil')
+                                            =t('camps:stats.edit')
+                                a.cursor-p(ng-click='removeCamp(camp.id)') &nbsp;
                                     button.btn.btn-xs 
```

Comment:
editing camp from the admin camps edit page.
use this unless we want to (sys) admin to be able to edit camps all the time